packages/airnode-node/src/workers/cloud-platforms/aws.ts

```diff
@@ -1,7 +1,7 @@
 import { logger } from '@api3/airnode-utilities';
 import { go } from '@api3/promise-utils';
 import { LambdaClient, InvokeCommand } from '@aws-sdk/client-lambda';
-import { Uint8ArrayBlobAdapter } from '@aws-sdk/util-stream';
+import { Uint8ArrayBlobAdapter } from '@smithy/util-stream';
```

Comment:
Is it worth adding a comment to say that this is an indirect dependency of `@aws-sdk/client-lambda`?